.claude/skills/auto-engineer/SKILL.md

```
@@ -0,0 +1,211 @@
+- No unresolved actionable review findings.
+- Either ≥1 review bot posted and every actionable finding is addressed, **or** 30 min elapsed since PR open with nothing actionable.
+
+Determine merge method from recent history:
+
+```sh
+gh pr list --state merged --limit 5 --json number,mergeCommit,title
+```
+
+Match the dominant style (squash is the current default — confirm before each merge). Then:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)